FreddysBBQ/src/AdminPortal/Program.cs

```diff
@@ -10,19 +10,17 @@ public class Program
     {
         public static void Main(string[] args)
         {
-            BuildWebHost(args).Run();
+            CreateHostBuilder(args).Build().Run();
         }
 
-        public static IWebHost BuildWebHost(string[] args) =>
-                WebHost.CreateDefaultBuilder(args)
-                        .UseCloudFoundryHosting()
-                        .AddCloudFoundry()
-                        .ConfigureLogging((builderContext, loggingBuilder) =>
-                        {
-                            loggingBuilder.AddConfiguration(builderContext.Configuration.GetSection("Logging"));
-                            loggingBuilder.AddDynamicConsole();
-                        })
-                        .UseStartup<Startup>()
-                        .Build();
+        public static IHostBuilder CreateHostBuilder(string[] args) =>
+            Host.CreateDefaultBuilder(args)
+                .AddCloudFoundryConfiguration()
+                .AddDynamicLogging()
+                .ConfigureWebHostDefaults(webBuilder =>
+                {
+                    webBuilder.UseStartup<Startup>();
+                })
+                .UseCloudHosting();
```

Comment:
We could set a port here to avoid conflict if both projects are started with `dotnet run`, but I don't think it's a required change